ksql-rest-app/src/main/java/io/confluent/ksql/rest/entity/ServerInfo.java

```diff
@@ -25,20 +25,39 @@
 import java.util.Objects;
 
 @JsonTypeInfo(include = JsonTypeInfo.As.WRAPPER_OBJECT, use = JsonTypeInfo.Id.NAME)
-@JsonTypeName("KSQL Server Info")
+@JsonTypeName("KsqlServerInfo")
 @JsonSubTypes({})
 public class ServerInfo {
   private final String version;
+  private final boolean uiEnabled;
+  private final String uiAddress;
 
   @JsonCreator
-  public ServerInfo(@JsonProperty("version") String version) {
+  public ServerInfo(@JsonProperty("version") String version,
+                    @JsonProperty("uiEnabled") boolean uiEnabled,
+                    @JsonProperty("uiAddress") String uiAddress
+  ) {
     this.version = version;
+    this.uiEnabled = uiEnabled;
+    if (!uiEnabled) {
+      this.uiAddress = "Not available. Enable by editing ksqlserver.properties / ui.enabled=true";
```

Comment:
Suggestion:
> "Not available. Enable by adding ui.enabled=true to the KSQL configuration file."

Reply:
It's a bit odd that `this.uiAddress` serves the dual-purpose of (1) tracking the actual UI address and (2) sth like a logging/help message.